js/src/forum/components/mapBBCode.js

```diff
@@ -7,7 +7,7 @@ export default function insertAtCursor(tilesProvider, style, zoom) {
     let pos = input.selectionStart
     let front = (input.value).substring(0, pos);
     let back = (input.value).substring(pos, input.value.length);
-    let middle = `[map provider=${tilesProvider??'osm'} style=${style??'street'} zoom=${zoom??13}]${app.translator.trans('jeromegillard-map.forum.text_editor.location_placeholder')}[/map]`;
+    let middle = `[map provider=${tilesProvider??'osm'} style=${style??'street'} zoom=${zoom??13} title='Place' desc='Updating...']${app.translator.trans('jeromegillard-map.forum.text_editor.location_placeholder')}[/map]`;
```

Comment:
It seem to only work when there are no single quotes `'`:

👎 Not working: `title='Place'`
👍 Working `title=Place`

Hence, how to provide complex values such as `desc=Hello quoted "world"` that could contain special chars like quotes `"` or even spaces ` `?

That is the main reason I need the location to be within the bbcode `[map] free text here[/map]`.  
Is there another way to support special chars within attributes of a bbcode?

Reply:
Ok, got it working on my end! I restored your default line, with translations included.

Somehow, in `BBCodesProvider.php` removing the `?` in the `div` part but not in the `[map]` solved the error I was having.